### PR TITLE
feat: add `meteor` completion spec

### PR DESCRIPTION
**Pull request by getninjaN**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
New spec for [meteor](https://www.meteor.com/)

**What is the current behavior? (You can also link to an open issue here)**
There is no spec for meteor

**What is the new behavior (if this is a feature change)?**
I went through the entire Meteor CLI and added them to the spec.

**Additional info:**
There already is a pull request (#1009) but that one only adds the npm part, so I figured I'd create the entire thing.

### CODE REVIEW

File: src/meteor.ts [hunk 1]
```diff
@@ -0,0 +1,1073 @@
...
+      loadSpec: "npm",
+    },
+    {
+      name: "help",
+      description: "Get help on meteor command line usage",
+      args: {
+        name: "command",
+        description: "Built-in command",
+        isOptional: true,
+        suggestions: [
```
Comment (fedeci) [inline src/meteor.ts:57]: Instead of manually adding suggestions here you can use the `help` template.

File: src/meteor.ts [hunk 2]
```diff
@@ -0,0 +1,1073 @@
+import { filepaths } from "@fig/autocomplete-generators";
+
+const examplesGenerator: Fig.Generator = {
+  script: "meteor create --list",
+  postProcess: (output) => {
+    return output.split("\n").map((example) => {
+      if (example.includes("github.com")) {
+        return { name: example.split(":")[0].trim() };
+      }
+    });
```
Comment (fedeci) [inline src/meteor.ts:10]: `.map` must always return some value. You can use `.reduce` or `.filter().map()` here.

File: src/meteor.ts [hunk 3]
```diff
@@ -0,0 +1,1073 @@
...
+            name: "command",
+            suggestions: [
+              "maintainers",
+              "recommend-release",
+              "change-homepage",
+              "list-organizations",
+              "members",
+            ],
+          },
+        },
```
Comment (fedeci) [inline src/meteor.ts:102]: These should be added as subcommands

File: src/meteor.ts [hunk 4]
```diff
@@ -0,0 +1,1073 @@
...
+          name: ["--port", "-p"],
+          description: "Port to listen on (instead of the default 3000)",
+          args: {
+            name: "port",
+            isOptional: true,
+            default: "3000",
+          },
+        },
+        {
+          name: ["--inspect", "--inspect-brk"],
```
Comment (fedeci) [inline src/meteor.ts:125]: In node these two flags are different, it seems it is the same for meteor

File: src/meteor.ts [hunk 5]
```diff
@@ -0,0 +1,1073 @@
...
+          description: "Create a basic blaze-based app",
+        },
+        {
+          name: "--tailwind",
+          description:
+            "Create a basic react-based app, with tailwind configured",
+        },
+        {
+          name: "path",
+        },
```
Comment (fedeci) [inline src/meteor.ts:288]: Is this a flag or an argument?

**Review by fedeci — DISMISSED** [review]
Thanks @getninjaN! I left some comments to fix before merging this spec!

**Comment by getninjaN** [comment]
> Thanks @getninjaN! I left some comments to fix before merging this spec!

Great! I'll take a look at it asap.

**Comment by getninjaN** [comment]
Changes has been made. 🍻

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**